### PR TITLE
InsertNode RemoveNode initial transforms

### PR DESCRIPTION
**Pull request by elm2178**

### CODE REVIEW

File: src/ot/Transformation.js [hunk 1]
```diff
@@ -1,4 +1,36 @@
...
+const removeNodeTransformHelper = (op1, removeNodeOp) => {
+  const newPath = PathUtils.transform(op1.get('path'), removeNodeOp).get(0);
+  if (newPath) {
+    return Operation.create({
+      ...op1.toJSON(),
+      path: newPath,
+    });
+  }
+
+  const nodeIsBeingRemoved =
```
Comment (qqwee) [inline src/ot/Transformation.js]: I think the PathUtils.transform already takes care of this step. So we can merge insertNodeTransformHelper and removeNodeTransformHelper together.

File: src/ot/Transformation.js [hunk 2]
```diff
@@ -557,21 +557,162 @@ const Transform = {
...
+    return op1;
+  },
+
+  /**
+   * [remove_node, insert_node] transformation.
+   * @param {Operation} op1
+   * @param {Operation} op2
+   * @param {String} side
+   */
+  transformRemoveNodeInsertNode: (op1, op2, side) => {
```
Comment (qqwee) [inline src/ot/Transformation.js]: Do you think we can also leverage the insert/removeNodeTrasformHelpers for these two cases:
transformRemoveNodeInsertNode and  transformRemoveNodeRemoveNode?

File: src/ot/Transformation.js (hunk 1 above)
Comment (elm2178) [reply src/ot/Transformation.js]: I'll take a look at this one, not sure if we can merge in

File: src/ot/Transformation.js (hunk 2 above)
Comment (elm2178) [reply src/ot/Transformation.js]: Yeah we probably can

**Pull request merged**